README.md

```diff
@@ -29,7 +29,7 @@ Add the following to your project:
 ```
 buildscript {
     dependencies {
-        classpath 'org.mozilla.android:apilint:0.1'
+        classpath 'org.mozilla.apilint:apilint:0.1'
```

Comment:
Is it a bad idea to just use `org.mozilla`?

Reply:
I'm actually not sure, I tried `org.mozilla` before but it seems to produce a weird folder structure in the maven repository. From [0], `groupId` is supposed to be unique per project, so `org.mozilla.apilint` would make sense. Maybe @ncalexan knows? :)

[0] https://maven.apache.org/guides/mini/guide-naming-conventions.html

Reply:
Yeah, we prefer to have the Maven group per-project.